contrib/container/Dockerfile

```diff
@@ -11,6 +11,7 @@
 
 ARG base_image=python:3.11-alpine3.18
 FROM ${base_image} AS inventree_base
+ARG base_image
```

Comment:
Why hvae you added a blank `base_image` arg here? Won't this null out the default value?

Reply:
No, it is a redefinition to carry the value over to this step from the previous one as ARGs are scoped per step. So the value as was definied in the previous step will be used.
https://docs.docker.com/reference/dockerfile/#scope